src/docker/Thinktecture.Relay.ManagementApi.Docker/appsettings.json

```diff
@@ -11,5 +11,16 @@
   "ConnectionStrings": {
     "PostgreSql": "host=relay_persistence_postgresql;database=relayserver;username=relayserver;password=<Strong!Passw0rd>"
   },
+  "Authentication": {
+    "ApiKey": {
+      "HeaderName": "tt-apikey",
```

Comment:
Please use a more conform header name, e.g. `X-TT-APIKEY`

Reply:
We can do uppercase, but the use of X- as prefix for custom headers is deprecated for the last decade ;)
https://www.rfc-editor.org/rfc/rfc6648

Reply:
To be in line with common headers (`Content-Encoding`, `Accept-Encoding` etc) I'd go for `TT-Api-Key` in our demo management API setup. Since this is configurable anyways a customer using that same system would probably chose their own header name prefixed with their company or project name / short name.